Demo/Features/Visa Checkout/BraintreeDemoVisaCheckoutViewController.m

```diff
@@ -7,7 +7,7 @@
 
 @interface BraintreeDemoVisaCheckoutViewController ()
 @property (nonatomic, strong) BTVisaCheckoutClient *client;
-@property (nonatomic) LaunchHandle launchHandler;
+@property (nonatomic, copy) LaunchHandle launchHandler;
```

Comment:
question: is it `copy` instead of `strong` here because `LaunchHandle` is a struct?

Reply:
It's actually a block. We made this change because this is what Visa Checkout has in their sample app, but according to this [SO post](https://stackoverflow.com/a/27156860), using `copy` with blocks is no longer needed.

Reply:
So we can make it `strong` instead.

Reply:
Aah nice aight. That's an awesome breakdown on the stack post.